frontend/libs/studio-components-legacy/src/components/StudioExpression/StudioExpression.test.tsx

```diff
@@ -258,6 +258,47 @@ describe('StudioExpression', () => {
     expect(onChange).not.toHaveBeenCalled();
     screen.getByText(texts.cannotSaveSinceInvalid);
   });
+
+  it('Displays all function types in the type selector by default', async () => {
+    const user = userEvent.setup();
+    renderExpression(logicalExpression);
+    await user.click(screen.getAllByRole('button', { name: texts.edit })[0]);
+    const typeSelector = screen.getAllByRole('combobox', { name: texts.valueType })[0];
```

Comment:
For meg som ikke er godt kjent med komponenten, hadde det vært enklere å forstå hvilke elementer dette er hvis det hadde vært brukt hjelpefunksjoner. Hva tenker du om noe slikt?
```suggestion
    await user.click(getFirstExpressionEditButton());
    const typeSelector = getFirstSubexpressionTypeSelector();
```

Reply:
Skjønner - ble veldig bra 🙂